src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.Production.yaml

```diff
@@ -33,6 +33,7 @@ config:
   edxapp:use_docker: true
   edxapp:web_node_capacity: "9"
   edxapp:worker_node_capacity: "5"
+  edxapp:worker_instance_type: "c6a.large"
```

Comment:
Should we set this as the default for all production deployments?

Reply:
They should all have similar usage. It's definitely worth running on non-burstable instances at the least.

Reply:
So actually it is interesting, mitxonline is the only even moderately taxed production environment. I would say the default could actually be t3a.medium. They all sit nicely at ~2.5 GB memory and full CPU balances. That would save a couple hundred $ a month.